src/org/labkey/targetedms/folderImport/PanoramaQCSettings.java

```diff
@@ -0,0 +1,478 @@
+                    List<Map<String, Object>> tsvData = loader.load();
+                    List<Map<String, Object>> dataWithoutDuplicates = new ArrayList<>();
+
+                    tsvData.forEach(row -> {
+                        String metricName = (String) row.get("MetricId");
+                        Integer metricId = getRowIdFromName(TargetedMSSchema.TABLE_QC_METRIC_CONFIGURATION, Set.of("Id"), new SimpleFilter(FieldKey.fromParts("Name"), metricName), ctx.getUser(), ctx.getContainer());
+                        row.put("MetricId", metricId);
+                        getReplicateDataWithoutDuplicates(ctx, ti, row, dataWithoutDuplicates);
+                    });
+                    return dataWithoutDuplicates.size() > 0 ? qus.insertRows(ctx.getUser(), ctx.getContainer(), dataWithoutDuplicates, errors, null, null).size() : 0;
+
+                }
+            },
+    QC_PLOT_SETTINGS (null, QCFolderConstants.QC_PLOT_SETTINGS_PROPS_FILE_NAME)
+            {
+                @Override
+                public void writeSettings(VirtualFile vf, Container container, User user)
+                {
+                    // Should get these settings if present:
+                    // Plot metric,
+                    // Date Range,
+                    // Plot size,
+                    // QC Plot type,
+                    // Y-Axis Scale
+                    // Plot Display Options:
+                    //-Group X-Axis Values by Date - checkbox value
+                    //-Show All Series in a Single Plot - checkbox value
+                    //-Show Excluded Points - checkbox value
+                    //-Show Reference Guide Set - checkbox value
+                    //-Show Excluded Precursors– checkbox value
+
+                    Map<String, String> plotSettings = PropertyManager.getProperties(user, container, QCFolderConstants.CATEGORY);
+
+                    try (PrintWriter out = vf.getPrintWriter(QCFolderConstants.QC_PLOT_SETTINGS_PROPS_FILE_NAME)) {
+
+                        Properties prop = new Properties();
+                        for (String name : plotSettings.keySet())
+                        {
+                            if (name.equalsIgnoreCase("metric"))
+                            {
+                                String metricValue = getMetricNameFromRowId(TargetedMSSchema.TABLE_QC_METRIC_CONFIGURATION, user, container, Integer.valueOf(plotSettings.get(name)));
+                                prop.put(name, metricValue);
+                            }
+                            else
+                            {
+                                prop.put(name, plotSettings.get(name));
+                            }
+                        }
+                        prop.store(out, null);
+
+                    }
+                    catch (Exception e)
+                    {
+                        LOG.error("Error exporting 'default view QC Plot settings'", e);
```

Comment:
Are you expecting errors here? Rethrowing is probably better to make sure we ultimately learn of them and adjust as needed